__tests__/app.test.js

```diff
@@ -325,3 +325,81 @@ describe("6. 09-GET_api_articles_article.id_comments /api/articles/:article_id/c
       });
   });
 });
+
+describe("8. 10-POST_api_articles_article.id_comments /api/articles/:article_id/comments", () => {
+  test("status:201, responds with an array of new comments for given test article_id objects", () => {

```

Comment:
I think your test description should be returns posted comment or something like that rather than an array of new comments. You would only ever get back one!